package.json

```diff
@@ -35,13 +35,18 @@
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.4.4",
+    "axios": "^0.19.0",
```

Comment:
Axios should be in dependencies because deployment requires it. 
This might be one reason why the app is unable to get deployed.